docs/prow/prow-jobs-autobump.md

```diff
@@ -0,0 +1,63 @@
+
+# Image autobump 
+
+This document provides an overview of autobump Prow jobs.  
+
+## Overview
+
+Generic-autobumper tool allows for automatic bump of Docker images in files to their latest versions. The images have to be specified as a Docker image URL.
```

Comment:
Please rephrase first Overview sentence to make it clear that tool is updating tags in OCI image urls, not the Docker images itself.